ValheimPlus/Configurations/ConfigurationExtra.cs

```diff
@@ -54,49 +54,59 @@ public static bool LoadSettings()
             return true;
         }
 
-        public static Configuration LoadFromIni(string filename)
+        public static Configuration LoadFromIni(IniData configdata)
         {
-            FileIniDataParser parser = new FileIniDataParser();
-            IniData configdata = parser.ReadFile(filename);
-
             Configuration conf = new Configuration();
+            conf.ConfigData = configdata;
+            conf.ConfigData.ClearAllComments();
             foreach (var prop in typeof(Configuration).GetProperties())
             {
                 string keyName = prop.Name;
                 MethodInfo method = prop.PropertyType.GetMethod("LoadIni", BindingFlags.Public | BindingFlags.Static | BindingFlags.FlattenHierarchy);
-
                 if (method != null)
                 {
                     var result = method.Invoke(null, new object[] { configdata, keyName });
                     prop.SetValue(conf, result, null);
                 }
             }
-
             return conf;
         }
 
+        public static Configuration LoadFromIni(string filename)
+        {
+            FileIniDataParser parser = new FileIniDataParser();
+            return LoadFromIni(parser.ReadFile(filename));
+        }
+
         public static Configuration LoadFromIni(Stream iniStream)
+        {
+            using (StreamReader iniReader = new StreamReader(iniStream))
+            {
+                FileIniDataParser parser = new FileIniDataParser();
+                return LoadFromIni(parser.ReadData(iniReader));
+            }
+        }
```

Comment:
Whitespace between methods

Reply:
will add it before `public static void LoadConfigurationFromStream(Stream iniStream)`